zenqt/ui/visualize/timeline.py

```diff
@@ -212,3 +212,4 @@ def jump_frame(self, frameid):
         self.stop_play()
         self.slider.setValue(frameid)
         zenvis.status['frameid'] = frameid
+        zenvis.core.set_curr_frameid(frameid)
```

Comment:
zhe shi sheng me? xiu fu linux slider bug?

Reply:
no. one bug one pr
just for record video